### PR TITLE
E2e jest

### PR DESCRIPTION
**Pull request by ramirobg94**
The main files are:

test/3_e2e/001-login.test.js
test/3_e2e/002-survey-creation.test.js

### CODE REVIEW

**Comment by ramirobg94** [comment]
e2e test running! we have to finish the mocha-jest migration to run the unit testing and split the commands like this [https://stackoverflow.com/questions/49613127/run-jest-tests-only-for-current-folder](https://stackoverflow.com/questions/49613127/run-jest-tests-only-for-current-folder)

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,19401 @@
+{
```
Comment (minotogna) [inline package-lock.json:1]: do not commit package-lock.json. we use yarn

File: package.json [hunk 2]
```diff
@@ -175,5 +180,15 @@
   },
   "lint-staged": {
     "{common,core,server,test,webapp}/{**,!(migrations)}/*.js": "eslint --cache --fix"
+  },
+  "jest": {
```
Comment (minotogna) [inline package.json:184]: let's have a jest config file

File: test/2_integration/005calculatedValueUpdate.test.js [hunk 3]
```diff
@@ -77,17 +75,17 @@ describe('Calculated value test', () => {
...
 
-  after(async () => {
+  afterAll(async () => {
     if (survey) {
       await SurveyManager.deleteSurvey(Survey.getId(survey))
     }
   })
 
-  it('Calculated value updated', async () => {
+  test('Calculated value updated', async () => {
```
Comment (minotogna) [inline test/2_integration/005calculatedValueUpdate.test.js:84]: jest also has it. what's the difference with test?

Comment (ramirobg94) [reply test/2_integration/005calculatedValueUpdate.test.js:84]: It is just an alias... so no difference
https://jestjs.io/docs/en/api.html#testname-fn-timeout

File: test/docker-compose.yml [hunk 4]
```diff
@@ -27,5 +27,4 @@ services:
       HEADLESS_CHROME: 'true'
       TAIKO_BROWSER_PATH: /usr/local/lib/node_modules/chromium/lib/chromium/chrome-linux/chrome
 
-    # Execute tests and migrations when postgres is finished:
-    command: ./test/scripts/test_start.sh
+    command: ./test/scripts/test_start.sh
```
Comment (minotogna) [inline test/docker-compose.yml:30]: restore new line at the end of file

File: test/run_docker_tests.sh [hunk 5]
```diff
@@ -16,4 +16,4 @@ docker-compose -f test/docker-compose.yml \
     up \
     --build \
     --abort-on-container-exit \
-    --exit-code-from arena_test
+    --exit-code-from arena_test
```
Comment (minotogna) [inline test/run_docker_tests.sh:19]: restore new line at the end of file

File: test/scripts/test_start.sh [hunk 6]
```diff
@@ -17,15 +17,14 @@ cp test/db/migrations/*.js server/db/migration/public/migrations
 cp test/db/migrations/sqls/*.sql server/db/migration/public/migrations/sqls
 
 # Run unit and integration tests
-yarn run npm-run-all server:migrate test
+yarn run npm-run-all server:migrate
```
Comment (minotogna) [inline test/scripts/test_start.sh:20]: old tests should still run

**Pull request closed without merging**